src/Core/Transformers/Message.ts

```diff
@@ -53,7 +54,9 @@ export default class MessageTransformer {
       deleteChannelMessages: (options?: ChannelLogsQueryOptions) =>
         message.channel
           .fetchMessages(options)
-          .then((messages) => message.channel.bulkDelete(messages))
+          .then((messages) => message.channel.bulkDelete(messages)),
+      getMentionedUsers: () => message.mentions.members,
+      hasMentionedUsers: () => !!message.mentions.members.first()
```

Comment:
This is just a suggestion so not blocking
```suggestion
      hasMentionedUsers: () => Boolean(message.mentions.members.first())
```